packages/experimental/Overflow/src/OverflowItem/OverflowItem.types.ts

```diff
@@ -0,0 +1,27 @@
+import type { ViewProps } from 'react-native';
+
+import type { OverflowItemChangeHandler } from '../Overflow/Overflow.types';
+
+export const overflowItemName = 'OverflowItem';
+
+export interface OverflowItemCoreProps {
```

Comment:
>OverflowItemCoreProps [](http://example.com/codeflow?start=17&length=21)

You could have this extend ViewProps. Then you wouldn't need to redefine onLayout.